quinn-udp/src/unix.rs

```diff
@@ -265,15 +265,16 @@ fn send(
                 }
                 io::ErrorKind::WouldBlock => return Err(e),
                 _ => {
-                    // Some network adapters do not support GSO. Unfortunately, Linux offers no easy way
-                    // for us to detect this short of an I/O error when we try to actually send
-                    // datagrams using it.
+                    // Some network adapters and drivers do not support GSO. Unfortunately, Linux
+                    // offers no easy way for us to detect this short of an EIO or sometimes EINVAL
+                    // when we try to actually send datagrams using it.
                     #[cfg(target_os = "linux")]
-                    if e.raw_os_error() == Some(libc::EIO) {
+                    if e.raw_os_error() == Some(libc::EIO) || e.raw_os_error() == Some(libc::EINVAL)
```

Comment:
Is there some spelling here that concisely avoids the duplication and allows us to use `libc::EIO | libc::EINVAL`? Maybe `if let Some(libc::EIO | libc::EINVAL) = e.raw_os_err() {`?

Reply:
Went with `Some(libc::EIO) | Some(libc::EINVAL)` because it's really hard for me not to read `|` between two integer constants as "bitwise OR".